tools/validate-buildenv.sh

```diff
@@ -521,7 +525,10 @@ function validate_pkgsrc_pkgs {
 # /opt/local/bin.
 #
 function validate_opt_tools {
-    if [[ ! -f /opt/tools/bin/pkgin ]]; then
+    # If we're using a modern release there's no need for /opt/tools
+    if [[ "$PKGSRC_RELEASE" == "2023Q4" ]]; then
```

Comment:
How will this work for future bumps?

Reply:
We can add it back if and when the time comes and we really feel it's necessary, but I'd really prefer we just keep the origin images updated and avoid what is effectively a hack in the future.  Using `/opt/tools` was an expedient option when trying to continue using images that were already many years EOL at the time, but it's better to just use the native package set where possible.

Reply:
I didn't mean a future restoration of `/opt/tools`.

Today you check for "2023Q4" and bail (which is good).  What happens for 2024Q4, 2025Q4, whatever future next?  Do you just keep adding checks there?

Reply:
Ah ok. Hopefully once we're fully over to 2023Q4, then for 2024Q4 and onwards we can just remove all of the support for `/opt/tools`, rather than just adding to the list here.

Reply:
Yeah... for older images (and components that are stuck needing them to build), which will not grow in numbers, one could invert the check (if not crusty old one) if need be.